.travis.yml

```diff
@@ -1,12 +1,12 @@
 sudo: required
-dist: trusty
+dist: xenial
 services:
   - docker
 language: scala
 scala:
   - 2.12.9
 jdk:
-  - oraclejdk8
+  - openjdk8
```

Comment:
ToL: At some point we'll need to update our docs to point to Oracle's OpenJDK pages instead of the Oracle Oracle JDK (not confusing, at all.)